### DISCUSSION TITLE
Future Problem of the Moviepy.

### DISCUSSION THREAD

**Discussion by SohamTilekar** [Zulko/moviepy#2085] [General]
Moviepy Is Not Updated to the latest version of the Python.
Many Function of the Moviepy Do Not Work in the Newer Python Version.

**Comment by Drinoter** [comment]
The command "pip install moviepy" is installing modern versions of various libraries that moviepy depends on, and this is a problem. The solution I found was to install Python 3.11 and start testing old libraries, which took me a lot of time. New users are going to have many problems.

**Reply by Drinoter** [reply]
pip install urllib3==1.26.15 tqdm==4.65.0 requests==2.28.2 proglog==0.1.10 Pillow==9.4.0 numpy==1.24.2 imageio==2.26.1 imageio_ffmpeg==0.4.8 idna==3.4 decorator==4.4.2 colorama==0.4.6 charset_normalizer==3.1.0 certifi==2022.12.7

**Reply by Drinoter** [reply]
Please, if anyone knows how to improve library versions, do so and post here because these are the ones I use but I know there is room for improvement.

**Reply by SohamTilekar** [reply]
There are To many Problems With Moviepy. Zulko Do Not Maintain it. There is Only one Way using the python 3.7 with Moviepy Dev2

**Comment by 2255ffs1** [comment]
sgsgs45sd4g5s4dgぱくラバニラサダウのブはとてもおいしそうなりんかマカョ♂♀⊙

**Reply by SohamTilekar** [reply]
What?

**Comment by keikoro** [comment]
I'm closing this as this discussion doesn't add anything to existing issue https://github.com/Zulko/moviepy/issues/1874.
